### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- created landing page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+    <!-- header -->
+
+    <!-- image slider -->
+    <section class="p-20 relative" style="top: 7rem">
+      <div class="slideshow rounded-lg overflow-hidden" style="height: 30rem">
+        <div class="slides fade">
+          <div class="slide-number">1 / 3</div>
+          <img
+            src="https://picsum.photos/id/1015/500/200"
+            style="width: 100%; object-fit: contain"
```
Comment (abhishektanwar) [inline index.html:122]: Please try to avoid inline styling.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+          />
+          <div class="slide-text">Caption One</div>
+        </div>
+
+        <div class="slides fade">
+          <div class="slide-number">2 / 3</div>
+          <img
+            src="https://picsum.photos/id/1016/500/200"
+            style="width: 100%; object-fit: contain"
```
Comment (abhishektanwar) [inline index.html:131]: Please try to avoid inline styling.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+          />
+          <div class="slide-text">Caption Two</div>
+        </div>
+
+        <div class="slides fade">
+          <div class="slide-number">3 / 3</div>
+          <img
+            src="https://picsum.photos/id/1039/500/200"
+            style="width: 100%; object-fit: contain"
```
Comment (abhishektanwar) [inline index.html:140]: Please try to avoid inline styling.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+          style="width: 10px; height: 10px"
+          class="dot"
+          onclick="currentSlide(2)"
+        ></span>
+        <span
+          style="width: 10px; height: 10px"
+          class="dot"
+          onclick="currentSlide(3)"
+        ></span>
+      </div>
```
Comment (abhishektanwar) [inline index.html:165]: Suggestion : Please try not to use JavaScript (onClick) here . Doing this using id's makes more sense. 
Also inline styling can be avoided here.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+        <a href="https://github.com/Ananyamadhu08" target="_blank">
+          <i class="fab fa-github mr-2 text-rose-700"></i>
+        </a>
+        <a href="https://twitter.com/AnanyaMadhu27" target="_blank">
+          <i class="fab fa-twitter mr-2 text-rose-700"></i>
+        </a>
+        <a
+          href="https://www.linkedin.com/in/ananya-madhu-74479b206/"
+          target="_blank"
+        >
```
Comment (abhishektanwar) [inline index.html:299]: Suggestion : It is good to use rel="noopener noreferrer" with target="_black" to improve security.

File: index.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+// Thumbnail image controls
+function currentSlide(n) {
+  slides((slideIndex = n));
+}
+
+function slides(n) {
+  var i;
+  var slides = document.getElementsByClassName("slides");
+  var dots = document.getElementsByClassName("dot");
+  if (n > slides.length) {
```
Comment (abhishektanwar) [inline index.js:18]: Using const or let would be better here.

**Review by abhishektanwar — APPROVED** [review]
Please check the suggestions. Code looks clean and good to merge.

File: index.js [hunk 7]
```diff
@@ -0,0 +1,41 @@
+var slideIndex = 1;
+slides(slideIndex);
+
+// Next/previous controls
+function plusSlides(n) {
```
Comment (VedanthB) [inline index.js:5]: can use arrow functions here.

**Review by VedanthB — APPROVED** [review]
oood job ananya, everything looks good to merge

File: index.html (hunk 1 above)
Comment (Ananyamadhu08) [reply index.html:122]: I have added inline styles so the classes from my component lib are overwritten , i wanted a few properties to overwrite the styles being applied from my component lib. As inline styles are more specific i have used them here

File: index.html (hunk 3 above)
Comment (Ananyamadhu08) [reply index.html:140]: same here

File: index.html (hunk 4 above)
Comment (Ananyamadhu08) [reply index.html:165]: Okay i will make the changes

File: index.js (hunk 6 above)
Comment (Ananyamadhu08) [reply index.js:18]: Okay will make the changes

File: index.js (hunk 7 above)
Comment (Ananyamadhu08) [reply index.js:5]: Okay wiil make the changes

File: index.html (hunk 5 above)
Comment (Ananyamadhu08) [reply index.html:299]: Okay will add the changes

**Pull request merged**